pkcs5/Cargo.toml

```
@@ -34,7 +34,8 @@ hex-literal = "0.3"
 [features]
 alloc = []
 3des = ["pbes2", "des"]
-des-insecure = ["pbes2", "des"]
+des-insecure = ["pbes2", "des", "ecb"]
+ecb = ["block-modes"]
```

I don't think it makes sense to conflate these. How about:

```suggestion
aes-ecb-insecure = ["block-modes"]
des-insecure = ["pbes2", "des"]
```